protos/protos/public/modeldb/DatasetVersionService.proto

```diff
@@ -209,6 +211,9 @@ message UpdateDatasetVersionAttributes {
 
 message GetDatasetVersionAttributes {
     string id = 1;
+    repeated string attribute_keys = 2;
+    bool get_all = 3;
```

Comment:
why was this required, lets not do extra stuff without discussing on zoom / slack